static/templates/invitation_failed_error.hbs

```diff
@@ -1,6 +1,13 @@
-<p>{{ error_message }}</p>
+<p id="err_msg">{{ error_message }}</p>
```

Comment:
These IDs are too generic; one could easily have e.g. CSS intended for this matching other error messages.

I'd call this `invitation_error_message`, and then one below `invitation_non_admin_message`.